optimade/validator/validator.py

```diff
@@ -451,22 +453,29 @@ def _get_archetypal_entry(
 
 
         """
-        response, _ = self._get_endpoint(endp)
-        data_returned = response.json()["meta"]["data_returned"]
-        if data_returned < 1:
-            return (
-                None,
-                "Endpoint {endp!r} has no entries, cannot get archetypal entry.",
+        response, message = self._get_endpoint(endp, multistage=True)
+
+        if response:
+            data = response.json().get("data", [])
+            data_returned = len(data)
+            if data_returned < 1:
+                return (
+                    None,
+                    "Endpoint {endp!r} returned no entries, cannot get archetypal entry.",
+                )
+
+            response, message = self._get_endpoint(
+                f"{endp}?page_offset={random.randint(0, data_returned-1)}&response_fields={','.join(properties)}",
```

Comment:
```suggestion
                f"{endp}?page_offset={random.randint(1, data_returned-1)}&response_fields={','.join(properties)}",
```

Reply:
What's wrong with page_offset being 0? We're only guaranteed that data_returned >= 1, not sure how well defined page_offset=1 is when there's only one result...

Reply:
It just seemed you wanted a non-zero value for `page_offset`, this will ensure it. But there's nothing wrong with it being `0` of course.

Reply:
I just want any entry from the endpoint, 0 is fine too